### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by ducfilan**
Add clarification for FamilyControls and fix sample code typo.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -35,7 +35,22 @@ yarn add react-native-screen-time-api
 Then run `npx pod-install`.
 
 ## Usage
+### Add FamilyControls capability to your app
```
Comment (NoodleOfDeath) [inline README.md:39]:
```suggestion

### Add FamilyControls capability to your app
```

File: README.md [hunk 2]
```diff
@@ -46,7 +61,7 @@ React.useEffect(() => {
     if (status !== 'approved') {
       throw new Error('user denied screen time access');
     }
-    await selection = await ScreenTime.displayFamilyActivityPicker();
+    const selection = await ScreenTime.displayFamilyActivityPicker();
```
Comment (NoodleOfDeath) [inline README.md:68]: nice catch

File: README.md [hunk 3]
```diff
@@ -35,7 +35,22 @@ yarn add react-native-screen-time-api
 Then run `npx pod-install`.
 
 ## Usage
+### Add FamilyControls capability to your app
+See https://developer.apple.com/documentation/Xcode/adding-capabilities-to-your-app
```
Comment (NoodleOfDeath) [inline README.md:40]:
```suggestion
See https://developer.apple.com/documentation/Xcode/adding-capabilities-to-your-app

In addition to adding the Family Controls entitlement, for distribution, you will also need to [request Family Controls capabilities](https://developer.apple.com/contact/request/family-controls-distribution)

```

**Review by NoodleOfDeath — APPROVED** [review]
Looks good, just added a few suggestions!

**Comment by ducfilan** [comment]
Thank you @NoodleOfDeath !
Just committed your suggestions.

**Comment by NoodleOfDeath** [comment]
Feel free to merge!

**Comment by ducfilan** [comment]
@NoodleOfDeath I think only you can merge.

**Comment by NoodleOfDeath** [comment]
Huh, you should be able to, but I'll go ahead and

**Pull request merged**